.github/workflows/publish.yml

```diff
@@ -21,4 +22,5 @@ jobs:
     - name: Push Stable Package
       run: |
         $fileName = (Get-ChildItem -Recurse -Filter *.nupkg | Select -Property Name).Name
-        nuget push signed/${{fileName}} -Source https://api.nuget.org/v3/index.json -ApiKey ${{ secrets.api_key }} -Verbosity detailed -NonInteractive
+        nuget push signed/$fileName -Source https://api.nuget.org/v3/index.json -ApiKey ${{ secrets.api_key }} -Verbosity detailed -NonInteractive
```

Comment:
I hope you've tested this and 100% sure it's going to work.

Reply:
I mean, I didn't test this exact call, I tested with a `Write-Output $fileName` to ensure the correct variable was output. `${{fileName}}` did not work. It seems the `${{variable}}` syntax is only for pipeline variables interpolation, not script variables.

Reply:
Also be mindful of the syntax difference between PowerShell and Bash (jobs running Ubuntu default to Bash script)

Reply:
Yup, this is why I overrode the default shell with `shell: pwsh` to allow the use of the powershell cmdlets.